projects/experimental/components/swipe-action/swipe-actions.style.less

```diff
@@ -0,0 +1,48 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+:host {
+    .scrollbar-hidden();
+    --t-action-gap: 24;
+    --t-actions-padding: 0.5rem;
+    --t-item-size: 44;
+
+    display: inline-flex;
+    width: calc(1px * var(--t-content-width));
```

Comment:
I have some doubts about it

`actions` is not absolute positioned (we need to know they real width to calculate scroll distance)
they have width and in a flex container they “eat up” part of the width of the main block, and `perspective-origin` moves

and in my opinion it is more convenient to set the dimensions in the main block rather than worry about the wrapper.

<img width="308" alt="Снимок экрана 2024-01-26 в 15 47 53" src="https://github.com/taiga-family/taiga-ui/assets/46284632/3516a9ec-8c55-4962-91ca-d243eb809d89">

Reply:
anyway I did some refactoring and made only one resize observer